packages/dev/docs/pages/releases/2021-02-16.mdx

```diff
@@ -0,0 +1,144 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved Picker, Tooltip, Combobox and updated a number of our aria hooks. Also check out our new components currently under construction including a Color package. A special thanks to all of our contributors for a very active new year.
```

Comment:
Nit, not sure if we want to have components with backticks, we've had them with and without in previous release notes:
```suggestion
description: After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved `Picker`, `Tooltip`, `ComboBox` and updated a number of our aria hooks. Also check out our new components currently under construction including a `Color` package. A special thanks to all of our contributors for a very active new year.
```

Reply:
probably not in the meta description